src/lib/components/topbar/topbar.scss

```
@@ -5,10 +5,15 @@
   background-color: $action-lighten-5;
   box-sizing: border-box;
   display: flex;
-  justify-content: flex-end;
+  justify-content: space-between;
```

O ideal é mander `flex-end` aqui.

Aí cria um modifier de TopBar (sugestão de nome: `topbar--with-info`) e aí quando existir a prop `children`, usando `classNames` você coloca esse modifier.

---

Manter `flex-end` aqui.

```suggestion
  justify-content: flex-end;
```